doc/holochain_101/src/bridging.md

```diff
@@ -56,28 +56,23 @@ pub fn handle_sample_function(some_param: String) -> ZomeApiResult<Address> {
     // do something here
 }
 
-define_zome! {
-    entries: []
-
-    init: || { Ok(()) }
-
-    validate_agent: |validation_data : EntryValidationData::<AgentId>| {
+#[zome]
+mod zome {
```

Comment:
suggest `mod my_zome {` or something that illustrates the ability to name the zome whatever ya want.